dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -32,16 +32,37 @@ agentAddress udp6:161
                                                  #  system + hrSystem groups only
 view   systemonly  included   .1.3.6.1.2.1.1
 view   systemonly  included   .1.3.6.1.2.1.25.1
-
                                                  #  Default access to basic system info
-{% if snmp_rocommunities %}
-{% for community in snmp_rocommunities %}
+
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
 rocommunity {{ community }}
 rocommunity6 {{ community }}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RW' %}
```

Comment:
>RW [](start = 44, length = 2)

Move this RW if branch into above for loop.
Like:
```
{% for community in SNMP_COMMUNITY %}
{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
{% elif SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
{% endif %}
{% endfor %}
```

Reply:
I was thinking it might be easier to have the RO and RW in seperate stanza's when looking at the snmpd.conf file.  
That is why I setup it up the way I did.  
Is there a particular reason you want it the other way?

Reply:
Per our conversion I leaving these as separate as it is easier to parse the snmpd.conf file for what is RO and what is RW.  Resolving this.